lib/common/schemas.c

```diff
@@ -631,6 +632,7 @@ free_schema(gpointer data)
     free(schema->name);
     free(schema->transform);
     free(schema->transform_enter);
+    free(schema);
```

Comment:
`free_schema()` didn't free... the... schema...

I've done worse but still, lol.